elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -108,12 +108,16 @@ export interface P2PClosePacket {
 	readonly reason?: string;
 }
 
+export interface PeerLists {
+	readonly blacklist: ReadonlyArray<P2PPeerInfo>;
+	readonly seeds: ReadonlyArray<P2PPeerInfo>;
+	readonly fixedlist: ReadonlyArray<P2PPeerInfo>;
+	readonly whitelist: ReadonlyArray<P2PPeerInfo>;
+	readonly previouslist: ReadonlyArray<P2PPeerInfo>;
+}
```

Comment:
except seedPeers, other property should be optional and handled in `optional` way